.template-helpers/variant-windowsservercore.md

```diff
@@ -0,0 +1,8 @@
+## `%%REPO%%:windowsservercore`
+
+This image is based on [Windows Server Core (`microsoft/windowsservercore`)](https://hub.docker.com/r/microsoft/windowsservercore/). As such, it only works in places which that image does, such as Windows 10 (with insider previews) or Windows Server 2016.
+
+For information about how to get "Docker on Windows" installed and configured, please see the relevant "Quick Start" guide provided by Microsoft:
```

Comment:
Indeed, that makes sense, but I'll wait to give @neilpeterson a chance to comment as well before I update. :heart:

(These guides are simple and awesome, so I'm glad they're a good place to link! :smile:)